src/main/java/DayTwo.java

```diff
@@ -2,21 +2,25 @@
 import java.util.List;
 
 public class DayTwo {
+    //can we remove this state?
```

Comment:
So we talked briefly about recursion, out of interest can you think of a way to refactor this code that does away with this instance variable?